### ISSUE TITLE
Errors on build and run

### ISSUE THREAD

**Issue by MarkMcDonald** [aspnet/AspNetCore.Docs#16827]
Followed all the steps attempting this I get a build error in IHubProtocol.d.ts when I attempt to debug/build:

TS2580	Build:Cannot find name 'Buffer'. Do you need to install type definitions for node? Try `npm i @types/node`.	C:\Work\Samples\SignalRWebPack\SignalRWebPack\node_modules\@microsoft\signalr\dist\esm\IHubProtocol.d.ts	136	

I'm presuming there is perhaps a step I need to run not mentioned, such as running the command suggested, or something to do with versions or external tools. 

And if I try npm run release in package manager console I get an error as follows:

 ERR! enoent ENOENT: no such file or directory, open 'C:\Work\Samples\SignalRWebPack\package.json'

The package.json file is actually located in C:\Work\Samples\SignalRWebPack\SignalRWebPack, the folder above just contains the solution file - so looks like package manager console is running from the context of the solution file perhaps.

npm -v =&gt; 6.13.6
node -v =&gt; 6.13.6

And I'm using Visual Studio 2019.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 47557f89-3a14-1139-0074-38d931b35faf
* Version Independent ID: 1cee43b4-b672-e480-3e26-72e08a6c2d77
* Content: [Use ASP.NET Core SignalR with TypeScript and Webpack](https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr-typescript-webpack?view=aspnetcore-3.1&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/signalr-typescript-webpack.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/signalr-typescript-webpack.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @ssougnez
* Microsoft Alias: **bradyg**

**Comment by MarkMcDonald** [comment]
Just spotted other comment on cd to project directory and following 'npm run release' it now all builds.

I may be missing something though as I'm not clear how to get the the webpack development mode watch to work - I tried running the build script through task runner explorer but wasn't seeing any updates as I changed the typescript.

**Comment by MarkMcDonald** [comment]
Okay to add a little more detail. 

If I run 'npm run release' the wwwroot is cleared and any updates I made appear, although in src nested under index.ts I have an index.js file which doesn't get updated - not sure why.

If I just use the watch I get a new main.[hash].js but all others remain and the index.js also isn't updated. 

So it just looks like an issues with the nested index.js - which I've removed for now. It looks like that is generated if I try and build solution - which I guess is perhaps something I shouldn't try but after nearly 20 years has become a bit of a habit.
